axelrod/tests/strategies/test_prober.py

```diff
@@ -232,24 +252,33 @@ class TestNaiveProber(TestPlayer):
     }
 
     def test_strategy(self):
-        # Randomly defects and always retaliates like tit for tat.
         self.first_play_test(C)
         # Always retaliate a defection
-        self.responses_test([D], [C] * 2, [C, D])
+        actions = [(C, D)] + [(D, D)] * 10
```

Comment:
Against an Alternator this test becomes stochastic because the strategy randomly defects against a cooperation. The test we're replacing just checked the first defection and not a set of future rounds. I'll change this to use a `MockPlayer` that starts with `C` and then `D` for the remaining rounds.